tests/phpunit/tests/Checker/Checks/Plugin_Readme_Check_Tests.php

```diff
@@ -192,24 +192,6 @@ public function test_run_without_error_mpl2_license() {
 		$this->assertEmpty( $errors );
 	}
 
-	public function test_run_with_errors_mpl1_license() {
```

Comment:
This has been removed as it was checking license value from plugin header. The same plugin is used in `test_run_with_invalid_mpl1_license` (in Plugin_Header_Fields_Check_Tests) for checking for invalid license.